ques19.cpp

```diff
@@ -0,0 +1,35 @@
+Time Complexity-O(n)
+Space Complexity-O(n) where 'n' is the amount
```

Comment:
Time is O(mn).. Rest all good

Reply:
Yes Yes i had to include the size of cost as i'll be processing the Dp array 'm' time which is the size of the vector. I'm sorry I will change that.